src/demo/sections/DataTable.vue

```diff
@@ -58,37 +52,56 @@ export default defineComponent({
   components: { DataTable, Checkbox },
   data() {
     const data: {
+      selected: any[],
       sortBy: string
       sortDir: DataTableColumnSort
       columns: DataTableColumn
       rows: DataTableRow
     } = {
+      selected: [],
       sortBy: 'memberCount',
       sortDir: 'desc',
       columns: [
         {
           label: 'Name',
-          value: 'name'
+          value: 'name',
+          search: true
```

Comment:
I wonder if opt out would result in less effort?

Reply:
Hm it might, I'm not really sure though - let's see how we end up using it